Cake.ArgumentHelpers.Tests/Cake.ArgumentHelpers.Tests.csproj

```
@@ -13,7 +13,8 @@
   </ItemGroup>
   <!-- Global packages -->
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="0.28.0" PrivateAssets="All" />
+    <PackageReference Include="Cake.Core" Version="2.2.0" PrivateAssets="All" />
+    <PackageReference Include="Cake.Common" Version="2.2.0" PrivateAssets="All" />

```

```suggestion
    <PackageReference Include="Cake.Common" Version="2.0.0" PrivateAssets="All" />
```

Does this addin need v2.2.0 specifically to work? If not, we should reference v2.0.0 to maximize compatibility with runners as per [best practices](https://cakebuild.net/docs/extending/addins/best-practices#cake-reference)

---

Good point, the change does not depend on version 2.2.0